### PR TITLE
[7.44.x]JBPM-9542  Ignore 4 unit tests of business-central-tests that related to Jira

### PR DESCRIPTION
**Pull request by bxf12315**
**Thank you for submitting this pull request**

**JIRA**: _(please edit the JIRA link if it exists)_ 

[JBPM-9542](https://issues.redhat.com/browse/JBPM-9542)

**referenced Pull Requests**: _(please edit the URLs of referenced pullrequests if they exist)_

<details>
<summary>
How to retest this PR or trigger a specific build:
</summary>

* <b>a pull request</b> please add comment: <b>Jenkins retest this</b>
 
* <b>a full downstream build</b> please add comment: <b>Jenkins run fdb</b>
  
* <b>a compile downstream build</b> please  add comment: <b>Jenkins run cdb</b>

* <b>a full production downstream build</b> please add comment: <b>Jenkins execute product fdb</b>

* <b>an upstream build</b> please add comment: <b>Jenkins run upstream</b>
</details>

### CODE REVIEW

**Comment by bxf12315** [comment]
@mareknovotny  The PR just ignore UT related to Jira, it wants to fix the FDB test failed.

**Comment by bxf12315** [comment]
Jenkins run fdb

File: business-central-tests/business-central-tests-rest/src/test/java/org/kie/wb/test/rest/functional/RepositoryIntegrationTest.java [hunk 1]
```diff
@@ -42,6 +43,7 @@ public static void createSpace() {
         createSpace(SPACE);
     }
 
+    @Ignore
```
Comment (mareknovotny) [inline business-central-tests/business-central-tests-rest/src/test/java/org/kie/wb/test/rest/functional/RepositoryIntegrationTest.java:46]: please add here a reason and jira URL you created for fixing the tests in future

File: business-central-tests/business-central-tests-rest/src/test/java/org/kie/wb/test/rest/functional/SpaceIntegrationTest.java [hunk 2]
```diff
@@ -75,6 +76,7 @@ public void testCreateMinimal() {
         testCreate(space);
     }
 
+    @Ignore
```
Comment (mareknovotny) [inline business-central-tests/business-central-tests-rest/src/test/java/org/kie/wb/test/rest/functional/SpaceIntegrationTest.java:79]: please add here a reason and jira URL you created for fixing the tests in future

File: business-central-tests/business-central-tests-rest/src/test/java/org/kie/wb/test/rest/functional/ProjectIntegrationTest.java [hunk 3]
```diff
@@ -60,6 +61,7 @@ public static void createRepository() {
         createSpace(SPACE);
     }
 
+    @Ignore
```
Comment (mareknovotny) [inline business-central-tests/business-central-tests-rest/src/test/java/org/kie/wb/test/rest/functional/ProjectIntegrationTest.java:64]: please add here a reason and jira URL you created for fixing the tests in future

File: business-central-tests/business-central-tests-rest/src/test/java/org/kie/wb/test/rest/functional/ProjectIntegrationTest.java [hunk 4]
```diff
@@ -95,6 +97,7 @@ public void testCreateProjectWithVersion() {
         createProject("projectWithoutVersion", null, GROUP_ID, VERSION);
     }
 
+    @Ignore
```
Comment (mareknovotny) [inline business-central-tests/business-central-tests-rest/src/test/java/org/kie/wb/test/rest/functional/ProjectIntegrationTest.java:100]: please add here a reason and jira URL you created for fixing the tests in future

**Review by mareknovotny — CHANGES_REQUESTED** [review]
`@Ignore` annotation must be followed by a reason why it is ignored. The standard here is the URL of jira explaining that and it is also a task for future to fix the test and remove `@Ignore`

**Review by mareknovotny — APPROVED** [review]

**Pull request merged**